FreeRTOS/FreeRTOS
# Update kernel submodule for unit test

Update kernel submodule for unit test

Description
-----------
Update kernel submodule for vTaskDelete function change.

Test Steps
-----------
CI unit test should pass.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
